tests/src/toCKB_typescript/tests/liquidation_signertimeout.rs

```diff
@@ -1 +1,208 @@
+use super::{Byte32, ToCKBCellData};
+use crate::*;
+use ckb_testtool::{builtin::ALWAYS_SUCCESS, context::Context};
+use ckb_tool::ckb_types::{
+    bytes::Bytes,
+    core::{TransactionBuilder, TransactionView},
+    packed::*,
+    prelude::*,
+};
+use ckb_tool::{ckb_error::assert_error_eq, ckb_script::ScriptError};
+use molecule::prelude::*;
+use int_enum::IntEnum;
 
+
+const MAX_CYCLES: u64 = 10_000_000;
+const PLEDGE_INVALID: i8 = 8;
+const LOT_SIZE_INVALID: i8 = 7;
+const TX_INVALID: i8 = 6;
+const ENCODING: i8 = 4;
```

Comment:
The error code and `ToCKBStatus` are duplicated here.
When the contract changes, it is hard to change it accordingly.
Consider move the definitions into a separated file, and use a softlink in the tests or make a rule in Makefile to copy it to tests.

Reply:
This change may affect many people's rebase.Maybe it is better to open another PR and unify the standard code fmt? @mkxbl

Reply:
`Error` `ToCKBStatus` `LotSize` `XChainKind`  are added to utils/types

Reply:
ok